### PR TITLE
[#1] Set up base KMM base project

### PR DESCRIPTION
**Pull request by phongvhd93**
- #1 

## What happened 👀

- Create a base project for KMM IC
- Update `CODEOWNER`
- Update `.gitignore`
## Insight 📝

Create a base project for the team to work on the KMM IC. 

## Proof Of Work 📹

Screenshot 2023-04-04 at 09 29 11

Screenshot 2023-04-04 at 10 20 33

### CODE REVIEW

File: shared/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            iosSimulatorArm64Test.dependsOn(this)
+        }
+    }
+}
+
+android {
+    namespace = "co.nimblehq.avishek.phong.kmmic"
+    compileSdk = 33
+    defaultConfig {
+        minSdk = 24
```
Comment (AVI5HEK) [inline shared/build.gradle.kts]: How about we use `21` as the min SDK? Hopefully, we'll never have to start a client project with this version in the future, but it'd be nice to have a working code in hand when needed. What do you think?

**Review by AVI5HEK — COMMENTED** [review]
We usually have a new line at the end of the file. But the auto-generated files don't have it. It might be very cumbersome to add a new line for each of these 22 files now. So how about we update these files whenever we work on them?

Also, any idea about the architecture and package management yet? Are we going to follow any template or sample codebase?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+* @phongvhd93 @AVI5HEK
```
Comment (markgravity) [inline .github/CODEOWNERS]: Please add me as code reviewer 🙏

**Review by markgravity — APPROVED** [review]

File: iosApp/Podfile
Comment (suho) [inline iosApp/Podfile:1]: @phongvhd93 Please check all files that are missing the new line at the end of the file

**Review by suho — CHANGES_REQUESTED** [review]

File: iosApp/Podfile
Comment (phongvhd93) [reply iosApp/Podfile:1]: I solved all the missing blank line at EOF

**Review by suho — APPROVED** [review]

**Pull request merged**